### ISSUE TITLE
1.12.2-1.8.4 : cannot use movement forward/backward keys to navigate journal

### ISSUE THREAD

**Issue by Febilian** [HellFirePvP/AstralSorcery#478]
I picked up playing with this mod again recently, and almost immediately I noticed that I had to use the scroll wheel once again(like in much earlier versions, where it was the only way to use the journal) to navigate into the different sections of the guidebook.

It's certainly playable, and isn't really causing anything more than a minor inconvenience, but the journal still _says_ that you should be able to use the alternative controls to scrolling when you mouse over the little star in the corner... so something isn't quite adding up here.

Just thought you might want to know in case it hasn't been reported already(although I didn't see it!). It is possible that it is a mod conflict of some kind(although I'm not using anything that I didn't regularly use before, it is possible that a mod updated and changed something), so if you have trouble reproducing this, I can provide a list of the mods I am using.

**Comment by Doomgull** [comment]
double clicking works. Otherwise check your keybindings. Rebinding w/s causes issues.

**Comment by Febilian** [comment]
I have not rebound either of those keys, but was not aware of double clicking either.

**Comment by HellFirePvP** [comment]
Well, you still have to hover over a specific cluster with your mouse, otherwise the logic won't know where to zoom towards.

**Comment by Febilian** [comment]
I coulda sworn I was doing that. I just double checked it again as well, and mousing over the cluster has no effect on whether the movement keys do anything to zoom in or out.

**Comment by Doomgull** [comment]
Need to figure out which mod is causing that, then, since it works with AS alone. Something is conflicting.

**Comment by Febilian** [comment]
Well, here's the _very odd_ thing... I just set up an instance with literally only the Forge version I was using, and Astral Sorcery, just to make sure before I started the age-old game of "plug every other mod back in one at a time until something breaks again".

It still won't work in a newly created world, with literally only AS installed.

Forge version 14.23.1.2580
AS version same as before(in the title)

Updating to Forge version 14.23.1.2583: no changes.
Dropping back to the recommended version of Forge(14.23.1.2555): does not work on account of AS needing a higher version of Forge than this.
Using the version of Forge AS requests, 14.23.1.2560: ...still no change.

I initially thought that it would be odd if this were specifically an issue with my operating system, but given that it _has_ actually proven to be the determining issue in other issues in the past, I am using macOS Sierra. Maybe that's a factor.

In case this also proves to be a factor, the java version I am using is 1.8.0_144. I'll check if there's an update for it.
